samples/wifi/sr_coex/README.rst

```diff
@@ -211,6 +211,22 @@ Use this command for Wi-Fi throughput only, Bluetooth LE throughput only, or con
 
 Use this command for concurrent Wi-Fi and Bluetooth LE throughput with coexistence enabled test.
 
+Change the build target as given below for the nRF7001 DK, nRF7002 EK and nRF7001 EK.
+Build target for nRF7001 DK:
+.. code-block:: console
+	nrf7001dk_nrf5340_cpuapp
+Build target for nRF7002 EK and nRF7001 EK:
+.. code-block:: console
+	nrf5340dk_nrf5340_cpuapp
+
+Add the following SHIELD options for the nRF7002 EK and nRF7001 EK.
+For nRF7002 EK:
+.. code-block:: console
+	-DSHIELD=nrf7002ek_nrf7002 -Dhci_rpmsg_SHIELD=nrf7002ek_nrf7002_coex
```

Comment:
Please add a blank line before and after the ``.. code-block:: console `` entries. Like so:

> For nRF7001 EK:
>
>  .. code-block:: console
>  (indented code here)
>
> The generated HEX file...

Reply:
Done